fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    fetch('https://jsonplaceholder.typicode.com/users', { method: 'GET' })
+      .then(response => response.json())
+      .then(users => {
+        this.setState({ users: users });
+        this.setState({ isLoading: false });
```

Comment:
Instead of calling `setState` twice, it is better to call it once and update both values at the same time.

Reply:
Right! Thanks!